composer.json

```diff
@@ -16,12 +16,12 @@
     },
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.1",
+        "symfony/framework-bundle": "~2.3",
         "misd/linkify": "~1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
-        "symfony/twig-bundle": "~2.1"
+        "symfony/twig-bundle": "~2.3,>=2.3.24"
```

Comment:
Just seen this that `2.3.4` is required for the Twig version. A `conflicts` entry for `twig/twig` would be better.